<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment> * This file is part of the OpenRPT report writer and rendering engine,
 * and is Copyright (c) 2001-2019 by OpenMFG LLC, d/b/a xTuple.  It
 * is licensed to you under the xTuple End-User License Agreement ("the
 * EULA"), the full text of which is available at www.xtuple.com/EULA.
 * While the EULA gives you access to source code and encourages your
 * involvement in the development process, this Package is not free
 * software.  By using this software, you agree to be bound by the
 * terms of the EULA.</comment>
 <class>PathEditor</class>
 <widget class="QDialog" name="PathEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stroke properties</string>
  </property>
  <layout class="QHBoxLayout">
   <item>
    <layout class="QVBoxLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="1">
        <widget class="QSpinBox" name="_sbWidth"/>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="_lbXend">
         <property name="text">
          <string>X end:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="_lblWidth">
         <property name="text">
          <string>Width:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="_lbYend">
         <property name="text">
          <string>Y end:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLineEdit" name="_leXend"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="_leYstart"/>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="_leXstart"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="_lbYstart">
         <property name="text">
          <string>Y start:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="_lbXstart">
         <property name="text">
          <string>X start:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLineEdit" name="_leYend"/>
       </item>
       <item row="2" column="3">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:6pt; font-style:italic;&quot;&gt;in inches &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="_gbStyle">
       <property name="title">
        <string>Style</string>
       </property>
       <layout class="QGridLayout">
        <item row="0" column="0">
         <widget class="QRadioButton" name="_rbSolidStyle">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../images/OpenRPTWrtembed.qrc">
            <normaloff>:/OpenRPT/images/line_solid.png</normaloff>:/OpenRPT/images/line_solid.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>80</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="_rbDashedStyle">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../images/OpenRPTWrtembed.qrc">
            <normaloff>:/OpenRPT/images/line_dashed.png</normaloff>:/OpenRPT/images/line_dashed.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>80</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="_rbDottedStyle">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../images/OpenRPTWrtembed.qrc">
            <normaloff>:/OpenRPT/images/line_dotted.png</normaloff>:/OpenRPT/images/line_dotted.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>80</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout">
     <item>
      <widget class="QPushButton" name="_btnAccept">
       <property name="text">
        <string>&amp;OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_btnCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="_btnColor">
       <property name="text">
        <string>Color...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>_leXstart</tabstop>
  <tabstop>_leYstart</tabstop>
  <tabstop>_leXend</tabstop>
  <tabstop>_leYend</tabstop>
  <tabstop>_sbWidth</tabstop>
  <tabstop>_rbSolidStyle</tabstop>
  <tabstop>_rbDashedStyle</tabstop>
  <tabstop>_rbDottedStyle</tabstop>
  <tabstop>_btnColor</tabstop>
  <tabstop>_btnAccept</tabstop>
  <tabstop>_btnCancel</tabstop>
 </tabstops>
 <resources>
  <include location="../images/OpenRPTWrtembed.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>_btnAccept</sender>
   <signal>clicked()</signal>
   <receiver>PathEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>PathEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
